autotest-agent/src/jvmMain/kotlin/com/epam/drill/test/agent/instrument/strategy/selenium/ChromeDevTool.kt

```diff
@@ -22,7 +22,7 @@ import com.epam.drill.test.agent.session.*
 import com.epam.drill.agent.transport.http.HttpResponseContent
 import com.epam.drill.common.agent.transport.AgentMessage
 import com.epam.drill.test.agent.instrument.TestSessionHeadersProcessor
-import com.epam.drill.test.agent.transport.DevToolsMessageSender
+import com.epam.drill.test.agent.jscoverage.DevToolsMessageSender
```

Comment:
DevToolsMessageSender is not just for javascript coverage - it also handles setting HTTP headers for Browser-based tests

Reply:
_TODO_ -  test this change on realworld-java-and-js-coverage app to ensure we do not break this feature